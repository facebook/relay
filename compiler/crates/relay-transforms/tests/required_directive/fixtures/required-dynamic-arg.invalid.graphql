# expected-to-throw
query EnabledUserQuery($action: RequiredFieldAction!) {
  me @required(action: $action) {
    id
  }
}
