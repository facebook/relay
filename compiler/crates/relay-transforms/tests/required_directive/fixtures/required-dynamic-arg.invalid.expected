==================================== INPUT ====================================
# expected-to-throw
query EnabledUserQuery($action: RequiredFieldAction!) {
  me @required(action: $action) {
    id
  }
}
==================================== ERROR ====================================
✖︎ Expected `action` argument to be a literal

  required-dynamic-arg.invalid.graphql:3:24
    2 │ query EnabledUserQuery($action: RequiredFieldAction!) {
    3 │   me @required(action: $action) {
      │                        ^^^^^^^
    4 │     id
